benchmark/bench_BoseHubbardMom12D2C.jl

```diff
@@ -0,0 +1,22 @@
+module BenchBoseHubbardMom1D2C
```

Comment:
Just to point out that your file name is called "bench_BoseHubbardMom**12D**2C.jl". It's amazingly high dimensional for the Bose-Hubbard model :D

Reply:
This will be resolved once we merge #32 and rebase this PR.

Reply:
I rebased. This file no longer exists.